### PR TITLE
[12.0][ADD] add account_bank_statement_import_bypass_check module

### PR DESCRIPTION
**Pull request by Tonow-c2c**
Import bank statement 2019-12-12 12-15

### CODE REVIEW

**Review by sebalix — APPROVED** [review]
Code review.

File: account_bank_statement_import_bypass_check/wizard/account_bank_statement_import.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        """
+        self.ensure_one()
+        currency_code, account_number, stmts_vals = self.with_context(
+            active_id=self.ids[0]
+        )._parse_file(base64.b64decode(self.data_file))
+        if self.check_journal:
+            try:
+                self._find_additional_data(currency_code, account_number)
+            except UserError as u_error:
+                if not self.check_journal:
```
Comment (jcoux) [inline account_bank_statement_import_bypass_check/wizard/account_bank_statement_import.py:38]: Are you sure of this test?
If your test line 32 is true, your test here can't be true.
`self.check_journal != not self.check_journal`

Or call of `_find_additional_data` change the value of `check_journal`.
In this case, can you add a little comment to explain this?

**Review by jcoux — CHANGES_REQUESTED** [review]

File: account_bank_statement_import_bypass_check/wizard/account_bank_statement_import.py (hunk 1 above)
Comment (Tonow-c2c) [reply account_bank_statement_import_bypass_check/wizard/account_bank_statement_import.py:38]: New comment on https://github.com/OCA/bank-statement-import/pull/243/files#diff-204ab1d34e5796f546481c0211664ef5R35-R36

**Review by jcoux — APPROVED** [review]

**Comment by Tonow-c2c** [comment]
@fclementic2c you that seems ok for you ?

**Pull request merged**